agent/osv_agent.py

```diff
@@ -218,7 +218,7 @@ def _run_command(command: list[str] | str) -> str | None:
     """
     try:
         output = subprocess.run(command, capture_output=True, text=True, check=False)
-    except subprocess.CalledProcessError as e:
+    except (subprocess.CalledProcessError, UnicodeDecodeError) as e:
```

Comment:
depending on where the exception is coming from it might be possible to force process the file by setting `errors=ignore` when decoding

Reply:
Correct, but we will not see the error. which might be necessary for debugging later !